TSC/core-and-hosted-projects.md

```diff
@@ -0,0 +1,538 @@
+
+## Applying for Adoption as a Hosted Project
+
+If you would like to propose that the Bytecode Alliance adopt your project as a new hosted project, file an issue against [the Bytecode Alliance governance repository](https://github.com/bytecodealliance/governance). The issue should enumerate each item marked as "Hosted Projects: Required" with a checkbox. Provide supplemental information, examples, and evidence that your project meets the associated requirement as bullet points below this checkbox. Here is an example issue:
+
+```md
+Proposing the adoption of `my-example-project` as a Bytecode Alliance
+hosted project.
+
+My Example project is a cool project. Here is some high level information
+about what it does. There should be a link to the repo here.
+
+### Hosted Project Requirements
+
+* [ ] Alignment with the Bytecode Alliance Mission
+  * My Example Project has alignment with the Bytecode Alliance mission
+    because it is all about security, efficiency, and modularity blah
+    blah blah
+* [ ] Code Review
+  * We follow the code review requirements, for example here are some
+    examples showing our review culture in practice:
+    * link 1
+    * link 2
+    * link 3
+* Etc...
+```
+
+The Bytecode Alliance Technical Steering Committee (TSC) will provide timely acknowledgement of the application. The TSC will then perform an in-depth review of the project, evaluate whether it meets the hosted project requirements, and will decide based on consensus whether to accept the project. Next, the TSC will reply in the issue with the results of the review and whether the project has been accepted. If it has been accepted, then the TSC will work with maintainers on the logistics of moving the repository to the `bytecodealliance` github organization.
+
+## Applying for Promotion from Hosted Project to Core Project
+
+Once the project maintainers have come to a consenus agreement regarding the promotion to a core project, they will initiate the process by creating an issue within the [the Bytecode Alliance governance repository](https://github.com/bytecodealliance/governance). The issue should enumerate each item marked as "Hosted Projects: Required" with a checkbox. Provide supplemental information, examples, and evidence that your project meets the associated requirement as bullet points below the checkbox. Here is an example issue:
+
+
+```md
+Proposing the promotion of `my-example-project` to a core project.
+
+My Example project is a cool project. Here is some high level information
+about what it does. There should be a link to the repo here.
+
+### Core Project Requirements
+
+* [ ] Alignment with the Bytecode Alliance Mission
+  * My Example Project has alignment with the Bytecode Alliance mission
+    because it is all about security, efficiency, and modularity, blah
+    blah blah
+* [ ] Changelog
+  * link to changelog
+* Etc...
```

Comment:
Even this is part of a template, this list is clearly labeled as _Core Project Requirements_, that sounds like a something that is supposed to be met before applying. If there are 'etc's that might be needed in addition to mission statement and changelog, maybe that should be spelled out.